modules/rhel-compute-updating.adoc

```diff
@@ -52,7 +52,7 @@ By default, the base OS RHEL with "Minimal" installation option enables firewall
 +
 [source,terminal,subs="attributes+"]
 ----
-# subscription-manager repos --disable=rhocp-4.13-for-rhel-8-x86_64-rpms \
+# subscription-manager repos --disable=rhocp-4.14-for-rhel-8-x86_64-rpms \
```

Comment:
The main branch should be for 4.16, right? If so, we need to update it as `4.15` here, and use `4.14` for 4.15 branch.

Reply:
Eventually when the 4.16 branch is created, it'll be based on main, that is true. However, until that branching happens, the main branch should only have changes made that apply to the latest existing branch, which is 4.15. 

So for this PR, the changes for 4.15 need to be made first in the main branch and then cherry-picked to 4.15. That's why I'm making the changes to `4.14` in these lines. Let me know if you have any questions about that, but this is mostly just the way the docs team manages its repository.

Reply:
Got it, it's ok for me if we're updating the docs in this way.

Reply:
Sounds good, thanks! Keeping the version numbers 4.14 in this PR for the 4.15+ doc change.